### PR TITLE
Add caption for githubteacher

### LINKED ISSUE #2: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @rogeryanda
- [ ] @jmedinacsu
- [ ] @js3263827
- [ ] @kayakkat
- [ ] @rjenkins5725
- [ ] @liam09
- [ ] @TJSaint
- [ ] @SpringsLW
- [ ] @IAmHughes
- [ ] @edelgadojr
- [ ] @Phat6
- [ ] @octowilliam
- [ ] @jrhixson2002
- [ ] @dannyreid
- [ ] @mgthompson2811
- [ ] @anshubhatia
- [ ] @rogeryanda
- [ ] @githubteacher
- [ ] @Phat6
- [ ] @dannyreid
- [ ] @jrhixson2002
- [ ] @js3263827
- [ ] @jmedinacsu
- [ ] @IAmHughes
- [ ] @liam09
- [ ] @rjenkins5725
- [ ] @edelgadojr
- [ ] @rogeryanda
- [ ] @octowilliam
- [ ] @TJSaint
- [ ] @SpringsLW
- [ ] @rtsinclair
- [ ] @rogeryanda
- [ ] @gmeyer39
- [ ] @rtsinclair
- [ ] @lkelly1
- [ ] @js3263827
- [ ] @githubteacher
- [ ] @githubteacher
- [ ] @kayakkat
- [ ] @jrhixson2002
- [ ] @jrhixson2002
- [ ] @gmeyer39
- [ ] @lgibbens

### PR DESCRIPTION
**Pull request by githubteacher**
TL;DR - One sentence summary of what's going on. 

## Tasks remaining
- [ ] Get a review

Link to issue or service now
This resolves #2 

Leading questions

### CODE REVIEW

**Review by kayakkat — CHANGES_REQUESTED** [review]
maybe change to say a different name...jeff is 😢

**Review by kayakkat — APPROVED** [review]
